source/features/github-actions-indicators.tsx

```diff
@@ -82,8 +130,18 @@ async function addIndicators(workflowListItem: HTMLAnchorElement): Promise<void>
 		return;
 	}
 
+	const svgTrailer = <div className="ActionListItem-visual--trailing m-auto"/>;
+	svgTrailer.style.display = 'flex';
+	svgTrailer.style.gap = '4px';
```

Comment:
I think the class is called `gap-2`. Fix as needed

```suggestion
	const svgTrailer = <div className="ActionListItem-visual--trailing m-auto d-flex gap-2"/>;
```

If you need inline style, you can use `style={{display: 'flex'}}` for example, but here it should not be needed.